spec/support/env_helper.rb

```diff
@@ -0,0 +1,49 @@
+module Danger
```

Comment:
I think the option I used in the initial commit where I have two helpers `spec/support/{gitlab,github}_helper.rb` with a common interface and then tag the specs with the api_provider is cleaner